.github/workflows/samples-kotlin-server.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
           - samples/server/petstore/kotlin-springboot-delegate
           - samples/server/petstore/kotlin-springboot-modelMutable
           - samples/server/petstore/kotlin-springboot-reactive
+          - samples/server/petstore/kotlin-springboot-request
```

Comment:
looks like your force push removed some of the changes I made.

can yo remove this line and put it in .github/workflows/sampels-kotlin-servser-jdk17.yaml instead?

Reply:
Sorry about that, didn't realize you pushed changed to my repository. Made a new commit for PR.